internal/pkg/consensus/expected.go

```diff
@@ -284,7 +254,8 @@ func (c *Expected) validateMining(
 		if err != nil {
 			return errors.Wrapf(err, "failed to read sector infos from power table")
 		}
-		valid, err := c.VerifyPoSt(c.postVerifier, allSectorInfos, uint64(sectorSize), blk.EPoStInfo.PoStRandomness, blk.EPoStInfo.PoStProof, blk.EPoStInfo.Winners, blk.Miner)
+		valid, err := c.VerifyPoSt(c.postVerifier, allSectorInfos, uint64(sectorSize), blk.EPoStInfo.PoStRandomness,
```

Comment:
blocking: `VerifyPoSt()` needs to take in the VRF digest, not the VRFProof, so you need to hash the VRFProof that you pass in here.

Reply:
Fixed